history/characters/340000_night_elf.txt

```diff
@@ -1,16 +1,52 @@
-#Dynasty 128000
+#Dynasty 128000 # Mar'alith
 340000={
-	name=Mar'alith
+	name=Pas'draen
 	dynasty=128000
-	dna="kinkbpfiibj"
+	dna="kpekbpgnfeg"
 	culture=night_elf religion=kaldorei_religion
 	martial=5 diplomacy=6 stewardship=7 intrigue=3 learning=8
 	add_trait=skilled_tactician add_trait=diligent add_trait=brave add_trait=patient add_trait=strategist
 	disallow_random_traits = yes
-	1.9.23={ birth=yes add_trait=creature_night_elf immortal_age=18 }
+	1.12.21={ birth=yes add_trait=creature_night_elf immortal_age=18 }
 	20.1.1={ add_trait=class_warrior_6 }
+	25.1.1={
+			add_spouse=340001
+			effect={
+					add_lover=340001
+			}
+	}
 	1059.11.25={ death=yes }
 }
+340001={
+	name=Natalia
+	female=yes
+	dynasty=128000
+	dna="kinkbpfiibj"
+	culture=night_elf religion=kaldorei_religion
+	martial=4 diplomacy=5 stewardship=6 intrigue=5 learning=8
+	add_trait=scholarly_theologian add_trait=honest add_trait=shrewd add_trait=diligent add_trait=chaste
+	disallow_random_traits = yes
```

Comment:
If we don't have enough lore info, I would remove it to make other (traits you added above won't be touched) character traits random.